stories/Molecules/Imagecaption/imagecaption.scss

```diff
@@ -0,0 +1,63 @@
+@import "../../assets/scss/variables";
+@import "../../assets/scss/breakpoints";
+
+/* image caption */
+.image-caption {
+    display: block;
+    border-bottom: 2px solid $color-black;
+    @include devicebreak(small) {
+        display: flex;
+        padding-bottom: 25px;
+    }
+    .image__caption__para {
+        padding-right: 0px;
+        padding-bottom: 15px;
+        margin-bottom: 15px;
+        line-height: 19.25px;
+        color: $color-gray-600;
+        font-size: $font-size-14;
+        border-bottom: 1px solid $color-gray-500;
+        @include devicebreak(small) {
+            margin: 0;
+            border: 0;
+            padding-bottom: 0;
+            line-height: 22px;
+            padding-right: 20px;
+            font-size: $font-size-16;
+        }
+    }
+}
+
+.image--author {
+    flex-shrink: 0;
+    position: relative;
+    margin-bottom: 15px;
+    @include devicebreak(small) {
+        margin-bottom: 0;
+        padding-left: 40px;
+    }
+    &:before {
+        content: "";
+        top: 0;
+        left: 0;
+        width: 0px;
+        height: 45px;
+        position: absolute;
+        background-color: #A9B1B7;
+        @include devicebreak(small) {
+            width: 1px;
+        }
+    }
+    .author__caption {
+        display: inline-block;
+        @include devicebreak(small) {
+            display: block;
+        }
+    }
+    .author--label {
+        margin-right: 5px;
+        @include devicebreak(small) {
+            margin-right: 0;
+        }
+    }
+}
```

Comment:
@Shubham-Sharma-77 Please use the variables for color, spacing and correct the indentation of file to 2 spaces

Reply:
@Shubham-Sharma-77 We can also use .author { &__caption} for the classes in scss and please don't use the nesting of classes if it is nor required like 'author__caption' in 'image--author'